route53_crossfeed_app.tf

```diff
@@ -2,51 +2,70 @@
 # Resource records that support Crossfeed site cloudfront endpoints and application.
 # ------------------------------------------------------------------------------
 
+###############################################################################
+# Look up the existing hosted zones for both cyber.dhs.gov and cisa.dhs.gov
+###############################################################################
+###############################################################################
+# Look up the existing hosted zones for both cyber.dhs.gov and cisa.dhs.gov
+###############################################################################
+data "aws_route53_zone" "cyber_dhs_gov" {
+  provider     = aws.route53resourcechange
+  name         = "cyber.dhs.gov"
+  private_zone = false
+}
+
+data "aws_route53_zone" "cisa_dhs_gov" {
+  provider     = aws.route53resourcechange
+  name         = "cisa.dhs.gov"
+  private_zone = false
+}
 # ------------------------------------------------------------------------------
 # Prod entries
 # ------------------------------------------------------------------------------
 
 # The hosted_zone_id for the below records comes from https://docs.aws.amazon.com/general/latest/gr/elb.html
 # (ALBs in us-gov-west-1 region)
-# A‑record alias for cyber‑hygeine.cisa.dhs.gov → your ALB
+# A‑record alias for cyber‑hygiene.cisa.dhs.gov → your ALB (IPv4)
 resource "aws_route53_record" "cyber_hygiene_prod_A" {
   provider = aws.route53resourcechange
 
+  name    = "cyber-hygiene.${data.aws_route53_zone.cisa_dhs_gov.name}"
+  type    = "A"
+  zone_id = data.aws_route53_zone.cisa_dhs_gov.zone_id
+
   alias {
-    name                   = "crossfeed‑prod‑1638162291.us‑gov‑west‑1.elb.amazonaws.com."
     evaluate_target_health = false
+    name                   = "crossfeed-prod-1638162291.us-gov-west-1.elb.amazonaws.com."
     zone_id                = "Z33AYJ8TM3BH4J"
   }
-  name    = "cyber‑hygiene.${data.aws_route53_zone.cisa_dhs_gov.name}"
-  type    = "A"
-  zone_id = data.aws_route53_zone.cisa_dhs_gov.zone_id
 }
```

Comment:
I think you said that the A record has already been created by `cisa.dhs.gov` by someone else, in which case there is no need to recreate it here.